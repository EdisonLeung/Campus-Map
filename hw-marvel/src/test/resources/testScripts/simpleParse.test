# Test a simple file parse with one node
LoadGraph simpleGraph simpleGraph.csv

ListNodes simpleGraph