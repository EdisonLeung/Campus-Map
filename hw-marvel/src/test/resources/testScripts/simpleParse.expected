# Test a simple file parse with one node
loaded graph simpleGraph

simpleGraph contains: a